gen/build_deploy/aws.py

```diff
@@ -485,10 +485,7 @@ def gen_templates(arguments):
     with logger.scope("Validating CloudFormation"):
         validate_cf(cloudformation)
 
-    return gen.Bunch({
-        'cloudformation': cloudformation,
-        'results': results
-    })
+    return ResultTuple(cloudformation, results)
```

Comment:
This PR changes `gen.build_deploy.azure.gen_templates()` to return a dict and changes its member lookup from `.` to `[]` (https://github.com/dcos/dcos/pull/956/files#diff-f4dc0083a55e6bc92a7e71ea4bd8d6e6R252). Make it return a `NamedTuple` for consistency?